documentation/src/main/asciidoc/topics/proc_os_configuring_clusters.adoc

```diff
@@ -0,0 +1,67 @@
+[id="os-configuring-clusters_{context}"]
+= Configuring Data Grid for OpenShift Clusters 
+
+[id="os_cluster_discovery"]
+== Configuring Cluster Discovery 
+
+Data Grid for OpenShift can use either the Kubernetes or DNS discovery mechanisms for clustering. These discovery mechanisms enable images to automatically join clusters.
+
+Data Grid for OpenShift templates and services use DNS by default. If you deploy Data Grid for OpenShift directly from the image or custom template, you must configure the appropriate discovery mechanism.
+
+[id="os_cluster_discovery_dns"]
+=== Configuring DNS_PING 
+
+.Procedure
+
+To configure the DNS discovery mechanism for clustering, do the following:
+
+. Set `openshift.DNS_PING` as the value for the `JGROUPS_PING_PROTOCOL` environment variable:
```

Comment:
The `JGROUPS_PING_PROTOCOL` variable is not applicable to 8.x images and this content is out of date.

Reply:
@ryanemerson Honestly, everything seems incorrect for ISPN 16.

Reply:
I agree. I didn't list everything here as I spoke with @dfennessy on Slack. I think we can close this PR.

Reply:
Hey, sorry about this. JDG-7445 was still open, and I didn't realize that these changes weren't needed for ISPN 16.